userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -148,6 +167,9 @@ Kill Bill Administrator (or Kill Bill Admin):: The person responsible for instal
 
 == O
 
+[[overdue_system]]
+Overdue System:: A Kill Bill system that defines the flow that the system must follow when an account is overdue (that is, has an unpaid balance). Also known as "Dunning".
```

Comment:
Maybe mention that it's different but related to Payment Retry system and also that there is NO default overdue system settings.

Reply:
A Kill Bill system -> A Kill Bill module? I don't know if we have some consistency already here?